pkg/framework/plugins/podlifetime/pod_lifetime.go

```diff
@@ -124,9 +125,7 @@ func (d *PodLifeTime) Deschedule(ctx context.Context, nodes []*v1.Node) *framewo
 	podutil.SortPodsBasedOnAge(podsToEvict)
 
 	for _, pod := range podsToEvict {
-		if !d.handle.Evictor().NodeLimitExceeded(nodeMap[pod.Spec.NodeName]) {
-			d.handle.Evictor().Evict(ctx, pod, evictions.EvictOptions{})
-		}
+		d.handle.Evictor().Evict(ctx, pod, evictions.EvictOptions{})
```

Comment:
`!d.handle.Evictor().NodeLimitExceeded(nodeMap[pod.Spec.NodeName])` condition is still needed here